packages/common/src/hardforks/shanghai.json

```diff
@@ -3,5 +3,5 @@
   "comment": "Next feature hardfork after the merge hardfork having withdrawals, warm coinbase, push0, limit/meter initcode",
   "url": "https://github.com/ethereum/pm/issues/356",
   "status": "Pre-Draft",
-  "eips": [4895, 3651, 3855, 3860]
+  "eips": [4895]
```

Comment:
They weren't activated for the 4844 devnet-3 and 3860 specifically was causing us to compute a different base transaction fee than geth.  Since Shanghai isn't final, it didn't feel like an issue for purposes of this branch.  When we merge this branch, we can add back if specific Shanghai EIPs have been finalized.

Reply:
yes we should restore the deletions since devenet-4/next devnet of eip4844 will be totally rebased on current withdrawals which almost certainly will have these EIPs in shanghai